rero_mef/query.py

```diff
@@ -60,8 +60,11 @@ def _default_parser(qstr=None):
     urlkwargs.add('q', query_string)
 
     # include deleted
-    deleted = request.args.get('deleted')
-    if not deleted:
-        search = search.filter('bool', must_not=[Q('exists', field='deleted')])
+    with_deleted = request.args.get('with_deleted')
+    if not with_deleted:
+        search = search.filter('bool', must_not=[
+            Q('exists', field='deleted'),  # no deleted MEF's
```

Comment:
Is it specific to MEF or for all authorities?

Reply:
For all without VIAF.